lib/common/strings.c

```diff
@@ -602,38 +602,52 @@ crm_strdup_printf(char const *format, ...)
 }
 
 int
-pcmk__split_range(const char *srcstring, char separator, char **start, char **end)
+pcmk__split_range(const char *srcstring, long long *start, long long *end)
 {
-    const char *seploc = NULL;
+    char *remainder = NULL;
+
+    *start = -1;
+    *end = -1;
```

Comment:
You probably want to do this after asserting the pointers aren't null :)

Reply:
Yeah, but wouldn't it be a lot more fun to leave it like this and just see what happens?